docs/topics/native/native-ios-integration.md

```diff
@@ -48,9 +48,14 @@ The resulting output:
 ```text
 init on <_NSMainThread: 0x600003bc0000>{number = 1, name = main}
 shared.SwiftExample
-deinit on <NSThread: 0x600003b9b900>{number = 7, name = (null)}
+deinit on <_NSMainThread: 0x600003bc0000>{number = 1, name = main}
 ```
 
```

Comment:
The list should show the two different ways of doing it, shouldn't it?
Now it seems that the second way is just an addition to the first one...

Reply:
No, this is actually a list of exceptions from the main rule about the main queue, like minor notes. so the second item is indeed an addition